cloudflare/data_source_zone_dnssec.go

```diff
@@ -0,0 +1,98 @@
+package cloudflare
+
+import (
+	"fmt"
+	"log"
+	"time"
+
+	"github.com/cloudflare/cloudflare-go"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+)
+
+func dataSourceCloudflareZoneDNSSEC() *schema.Resource {
+	return &schema.Resource{
+		Read: dataSourceCloudflareZoneDNSSECRead,
+
+		Schema: map[string]*schema.Schema{
+			"zoneid": {
+				Type:             schema.TypeString,
+				Required:         true,
+				DiffSuppressFunc: zoneDiffFunc,
+			},
+			"status": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"flags": {
+				Type:     schema.TypeInt,
+				Computed: true,
+			},
+			"algorithm": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"key_type": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"digest_type": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"digest_algorithm": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"digest": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"ds": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"key_tag": {
+				Type:     schema.TypeInt,
+				Computed: true,
+			},
+			"public_key": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"modified_on": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
```

Comment:
I think we can do without the `modified_on` as we can't set or use it in other resources.

```suggestion
```

Reply:
fixed